app/subscriber/src/components/sidebar/hooks/useFilterOptions.ts

```diff
@@ -0,0 +1,14 @@
+import { useLookup } from 'store/hooks';
+
+export const useFilterOptions = () => {
+  const [{ sources }] = useLookup();
+  const dailyPapers = sources.filter(
```

Comment:
could we move these values to a array const perhaps?  Also, do these special values exist in editor or subscriber settings anywhere?